modules/roles_profiles/manifests/profiles/gpu_drivers.pp

```diff
@@ -3,17 +3,15 @@
 # file, You can obtain one at http://mozilla.org/MPL/2.0/.
 
 class roles_profiles::profiles::gpu_drivers {
-
-    case $::operatingsystem {
-        'Windows': {
-
-            class { 'win_packages::drivers::nvidia_grid':
-                driver_name => '391.81_grid_win10_server2016_64bit_international',
-                srcloc      => lookup('windows.s3.ext_pkg_src'),
-            }
-        }
-        default: {
-            fail("${::operatingsystem} not supported")
-        }
+  case $facts['os']['name'] {
+    'Windows': {
+      class { 'win_packages::drivers::nvidia_grid':
+        driver_name => '391.81_grid_win10_server2016_64bit_international',
```

Comment:
Just a note and related to this PR. No immediate action needed. we should move ` driver name` into hiera. This can be updated when we address Win 11 GPU workers.